app/controllers/employment.py

```diff
@@ -253,6 +253,7 @@ def mutate(cls, _, info, **employment_input):
                         "Invalid user id", should_alert_team=False
                     )
             if user_email:
+                user_email = user_email.lower()
                 user = (
                     User.query.options(selectinload(User.employments))
                     .filter(func.lower(User.email) == func.lower(user_email))
```

Comment:
:green_circle: Il n'est plus nécessaire de lower `user_email` dans la query : 
```suggestion
                    .filter(func.lower(User.email) == user_email)
```